### PR TITLE
[WIP] Postprocessor example

### PR DESCRIPTION
**Pull request by AlekseySh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -453,6 +449,91 @@ trainer.fit(pl_model)  # we don't pass loaders to .fit() in DDP
 [comment]:lightning-ddp-end
 </p>
 </details>
+ㅤ
+
+#### Postprocessing
+
+You can also boost retrieval accuracy of your features extractor by adding a postprocessor (we recommend
```
Comment (DaloroAT) [inline README.md:456]: is it ok to use `retrieval accuracy`?

File: oml/inference/flat.py [hunk 2]
```diff
@@ -49,16 +49,16 @@ def _apply(model_: nn.Module, batch_: Dict[str, Any]) -> Tensor:
...
-    dataset_root: Path,
+    dataset_root: Union[Path, str],
     dataframe_name: str,
-    output_cache_path: Optional[Path],
     extractor: IExtractor,
     transforms_extraction: TTransforms,
-    num_workers: int,
-    batch_size: int,
-    use_fp16: bool,
+    output_cache_path: Optional[Path] = None,
```
Comment (DaloroAT) [inline oml/inference/flat.py:56]: Union[Path, str]

**Review by DaloroAT — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AlekseySh) [reply README.md:456]: yes, I gust generalised the term

File: oml/inference/flat.py (hunk 2 above)
Comment (AlekseySh) [reply oml/inference/flat.py]: done
